lib/bolt_ext/server.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 
 require 'sinatra'
+# require 'sinatra/base'
```

Comment:
What's all the extra Sinatra stuff here?

Reply:
I was hoping to have time to figure out why I can't use 'namespace', and perhaps use it. It seems appropriate here and would be cleaner, but isn't worth sinking too much time into. If it's not done when the API docs get merged I'll remove the comments.